pkg/logevents/logevents_sql.go

```diff
@@ -0,0 +1,85 @@
+package logevents
+
+import (
+	"github.com/armory/dinghy/pkg/database"
+	"strings"
+	"time"
+)
+
+type LogEventSQLClient struct {
+	MinutesTTL	time.Duration
+	SQLClient 	*database.SQLClient
+}
+
+
+func (LogEventSQL) TableName() string {
+	return "logevents"
+}
+
+type LogEventSQL struct {
+	Id 			int		`gorm:"primaryKey;column:id"`
```

Comment:
Might wanna run this PR through `go fmt` before finalized so we avoid stuff like this

Reply:
Wtf didn't know go can do that. 😮